class.jetpack-gutenberg.php

```diff
@@ -689,4 +689,46 @@ public static function load_independent_blocks() {
 			}
 		}
 	}
+
+	/**
+	 * Get CSS classes for a block.
+	 *
+	 * @since 7.7.0
+	 *
+	 * @param string $slug  Block slug.
+	 * @param array  $attr  Array containing the block attributes.
+	 * @param array  $extra Array of potential extra classes you may want to provide.
+	 *
+	 * @return string $classes List of CSS classes for a block.
+	 */
+	public static function block_classes( $slug = '', $attr, $extra = array() ) {
+		if ( empty( $slug ) ) {
+			return '';
+		}
+
+		// Basic block name class.
+		$classes = array(
+			'wp-block-jetpack-' . $slug,
+		);
+
+		// Add alignment if provided.
+		if (
+			! empty( $attr['align'] )
+			&& in_array( $attr['align'], array( 'left', 'center', 'right', 'wide', 'full' ), true )
+		) {
+			array_push( $classes, 'align' . $attr['align'] );
+		}
+
+		// Add custom classes if provided in the block editor.
+		if ( isset( $attr['className'] ) ) {
```

Comment:
Not super important but this could have `! empty()`  check, too.

Reply:
That works for me. `! empty()` checks if it is set already so that's even better. Done in 6694fae6a906db543b28fb0bcc5c5822351865b6